parsing/parser.mly

```diff
@@ -1204,7 +1203,7 @@ open_statement:
         Opn.mk $4 ~override:$2 ~attrs:(attrs@$5) ~loc:(make_loc $sloc) ~docs
         , ext }
 ;
-sig_include_statement:
+%inline sig_include_statement:
```

Comment:
As for `structure_item`, I would vote for inlining none of the legitimate children of `signature_item{,_with_ext}` for consistency.